src/saltext/vault/pillar/vault.py

```diff
@@ -67,19 +89,22 @@
 import salt.utils.dictupdate
 import saltext.vault.utils.vault as vault
 import saltext.vault.utils.vault.helpers as vhelpers
+from salt.exceptions import InvalidConfigError
 from salt.exceptions import SaltException
+from saltext.vault.utils.versions import warn_until
 
 log = logging.getLogger(__name__)
 
 
 def ext_pillar(
     minion_id,  # pylint: disable=W0613
     pillar,  # pylint: disable=W0613
-    conf,
+    path=None,
     nesting_key=None,
     merge_strategy=None,
     merge_lists=None,
     extra_minion_data=None,
+    conf=None,
```

Comment:
**question (blocking):** we generally don't want to reorder parameters since it could affect people using the Python API without specifying the parameter names. is there a way to accomplish this?

Reply:
Good catch! I thought I had accounted for the positional arg case – and I did (it's also in the test), but it's not treated exactly like before, so might break if people have a weird config string that was accepted by the logic before.